### PR TITLE
Updates to support doc

### PR DESCRIPTION
**Pull request by dovholuknf**

### CODE REVIEW

**Comment by dovholuknf** [comment]
preview the experience by going to https://dovholuknf.github.io/ziti-doc/api/index.html and clicking on the c api

File: programs/sample_wttr/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 project(sample_wttr)
 
-add_executable(sample_wttr sample_wttr.c)
+add_executable(sample_wttr sample_wttr.c ${ZITI_HEADER_FILES})
```
Comment (ekoby) [inline programs/sample_wttr/CMakeLists.txt]: pretty sure that is not needed

File: building.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+\page building Building the Project
+
+The following steps should get your C SDK for Ziti building. C development is specific to your operating system and 
+tool chain used. These steps should work properly for you but if your OS has variations you may need to adapt these steps accordingly.
+
+[TOC]
```
Comment (ekoby) [inline building.md]: this shows up as literal [TOC] on the main project page... ugh

File: programs/ziti-prox-c/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 project(ziti-prox-c)
 
-add_executable(ziti-prox-c proxy.c)
+add_executable(ziti-prox-c proxy.c ${ZITI_HEADER_FILES})
```
Comment (ekoby) [inline programs/ziti-prox-c/CMakeLists.txt]: not needed?

Comment (dovholuknf) [reply programs/ziti-prox-c/CMakeLists.txt]: it's not needed for things to compile properly but without this entry the header files are buried in visual studio. This pic shows before the change (left) and after (right)

image

File: README.md [hunk 4]
```diff
@@ -1,52 +1,21 @@
...
-The following steps should get your C SDK for Ziti building. C development is specific to your operating system and 
-tool chain used. These steps should work properly for you but if your OS has variations you may need to adapt these steps accordingly.
+## Ziti C SDK
 
-## Status 
-[![Build Status](https://travis-ci.org/netfoundry/ziti-sdk-c.svg?branch=master)](https://travis-ci.org/netfoundry/ziti-sdk-c)
-
-## Prerequisites
-
-### Build
```
[the remark this answers is not in the document]
Comment (dovholuknf) [reply README.md]: sure - i forgot to do that.

File: programs/ziti-prox-c/CMakeLists.txt (hunk 3 above)
Comment (ekoby) [reply programs/ziti-prox-c/CMakeLists.txt]: still seems incorrect, an application that uses SDK won't add that or any other dependency headers (libuv, etc)

**Pull request closed without merging**